README.md

```diff
@@ -9,9 +9,7 @@ This release is made with the following assumptions:
 - you're using the [node_exporter](https://github.com/bosh-prometheus/node-exporter-boshrelease) bosh release to export stats to prometheus
 - node_exporter is loading /var/vcap/data/node_exporer/config/.&ast;.prom
 
-To use this release, install it, then schedule `run-report.sh` via cron. You probably want to use the [cron boshrelease](https://github.com/cloudfoundry-community/cron-boshrelease)
-for this, but nobody's gonna force you to. [RedHat](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/sec-using-aide)
- says you should run AIDE at least weekly, but not more than daily.
+The release will schedule AIDE to check and report hourly, by dropping a file into `/etc/cron.hourly`.
```

Comment:
Your note about running AIDE weekly vs. daily vs. hourly had me digging into this a bit more, and I couldn't find much else that spoke about this.  The Red Hat guide does say "at most" but gives no specific reason why.  I suspect it has to do with performance of the file system given the extra monitoring, and [this community Q&A response](https://access.redhat.com/discussions/779413#comment-736893) speaks to that somewhat:

>I think executing frequently using cron (to email + log) and re-baseline as part of patching or software deployment is a good basic approach. What you suggest regarding protection of the aide database sounds like a nice addition too. I would also suggest that if you are going to add it to cron to use a random sleep at the start of the job because AIDE can tax the filesystem during the check and you don't want all your VM's hitting the disk at the same time!

**Provided this change won't severely impact our system performance/stability,** I'm in favor of it as I agree with you that 24 hours is too long to go without a check of this nature.  I'd like to make sure we have a high degree of confidence in the security and integrity of our systems.

I also found what appears to be the most recent documentation in AIDE's GitHub repository, and [the section on how to run AIDE checks (`aide --check`)](https://github.com/aide/aide/blob/master/doc/manual.html#L501-L520) may also be helpful in making sure we have a reasonable configuration set for our checks.

Reply:
We had the same thoughts about a month ago.  We don't see why we shouldn't run it more often.  +1 to hourly, and we'll see if it explodes.